### PR TITLE
Add AnimatableScrollView

### PR DESCRIPTION
**Pull request by tbaranes**

### CODE REVIEW

File: IBAnimatable/AnimatableScrollView.swift [hunk 1]
```diff
@@ -0,0 +1,154 @@
+//
+//  Created by Tom Baranes on 28/06/16.
+//  Copyright © 2015 Jake Lin. All rights reserved.
+//
+
+
+import UIKit
+import IBAnimatable
```
Comment (TheDarkCode) [inline IBAnimatable/AnimatableScrollView.swift]: Why are you importing IBAnimatable?

Comment (tbaranes) [inline IBAnimatable/AnimatableScrollView.swift]: My bad. I did that in my project then copy in IBAnimatable, forgot to remove it.

**Comment by JakeLin** [comment]
👍 thanks, @tbaranes , I think we can release 2.4 once I fix some mask issues.

**Comment by tbaranes** [comment]
Agree for the release 👌

**Pull request merged**